### PR TITLE
Update cri-tools to v1.18.0

### PR DESCRIPTION
**Pull request by saschagrunert**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

Update cri-tools to the latest version and adapt test scripts.

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

cc @feiskyer 

**Does this PR introduce a user-facing change?**:

```release-note
- Update cri-tools to v1.18.0
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
- Release Notes: https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.18.0
```

/sig release
/cc @kubernetes/release-engineering

### CODE REVIEW

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -1242,13 +1242,14 @@ function DownloadAndInstall-Crictl {
...
-      'crictl-' + $CRICTL_VERSION + '-windows-amd64.exe')
+  $url = ('https://github.com/kubernetes-sigs/cri-tools/releases/download/' +
+          $CRICTL_VERSION + '/crictl-' + $CRICTL_VERSION + '-windows-amd64.tar.gz')
   MustDownload-File `
       -URLs $url `
-      -OutFile ${env:NODE_DIR}\crictl.exe `
+      -OutFile ${env:NODE_DIR}\crictl.tar.gz `
       -Hash $CRICTL_SHA256 `
       -Algorithm SHA256
+  tar xzvf ${env:NODE_DIR}\crictl.tar.gz
```
Comment (feiskyer) [inline cluster/gce/windows/k8s-node-setup.psm1:1252]: do we need to set dir to NODE_DIR for tar?

Comment (saschagrunert) [reply cluster/gce/windows/k8s-node-setup.psm1:1252]: I'll change that, thank you for the hint :+1:

**Review by feiskyer — APPROVED** [review]
/lgtm

**Review by fejta — COMMENTED** [review]
/approve

**Comment by saschagrunert** [comment]
@justaugustus May I ask you to approve if this is fine? Not sure if we need top level approved rights here.

**Comment by BenTheElder** [comment]
you can check who has permissions by clicking through to the OWNERS files in the bot comment :upside_down_face: 
/assign

**Comment by justaugustus** [comment]
Thanks Ben! :heart:

**Comment by saschagrunert** [comment]
Rebased on top of the latest master to resolve the merge conflict

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by justaugustus** [comment]
@mm4tt @shyamjvs @wojtek-t @kubernetes/sig-scalability-bugs -- This is pretty consistently failing on `pull-kubernetes-e2e-gce-100-performance`. Can you take a look?

/test pull-kubernetes-e2e-gce-100-performance
cc: @kubernetes/ci-signal

**Comment by saschagrunert** [comment]
I'm really wondering if it's related to my change :thinking: The build log does not seem to state anything related to a failing cri-tools download or such...

**Comment by BenTheElder** [comment]
/hold
https://prow.k8s.io/pr-history/?org=kubernetes&repo=kubernetes&pr=89720
https://prow.k8s.io/job-history/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-gce-100-performance

the gce-100-performance job is passing on other PRs but has only failed on this one.

**Comment by saschagrunert** [comment]
Solved the issue, PTAL @BenTheElder

**Comment by justaugustus** [comment]
@saschagrunert -- What was the fix?

File: cluster/gce/gci/configure.sh [hunk 2]
```diff
@@ -293,10 +293,10 @@ EOF
   fi
 
   echo "Downloading crictl"
-  local -r crictl_path="https://storage.googleapis.com/kubernetes-release/crictl"
+  local -r crictl_path="https://github.com/kubernetes-sigs/cri-tools/releases/download/${crictl_version}"
   download-or-bust "${crictl_sha1}" "${crictl_path}/${crictl}"
-  mv "${KUBE_HOME}/${crictl}" "${KUBE_BIN}/crictl"
-  chmod a+x "${KUBE_BIN}/crictl"
+  tar xf "${crictl}"
+  mv crictl "${KUBE_BIN}/crictl"
```
Comment (saschagrunert) [inline cluster/gce/gci/configure.sh:299]: @justaugustus this part had to be refactored to just untar the downloaded tarball and move the binary from the correct source location.

Comment (justaugustus) [reply cluster/gce/gci/configure.sh:299]: Thanks!
/lgtm
/approve
/hold cancel

**Pull request merged**